src/internet-computer-programming-concepts/basic-memory-persistence/upgrades.md

```diff
@@ -14,15 +14,18 @@ There are two ways to *update* the Wasm module of an actor running in a canister
 ### Reinstall
 When we have an actor Wasm module running in a canister, we could always reinstall that same actor module or a new actor module inside the same canister. Reinstalling always causes the actor to be 'reset'. Whether reinstalling the same actor Wasm module or a new one, the operation is the same as if installing for the first time.
 
-**Reinstalling always wipes out the main memory of the canister and erases the state of the actor including any stable variables.** 
+**Reinstalling always wipes out the main memory of the canister and erases the state of the actor** 
 
 ### Upgrade
 We could also choose to *upgrade* the Wasm module of an actor. Then:
 - [Stable variables](/internet-computer-programming-concepts/basic-memory-persistence/stable-variables.html) are persisted
```

Comment:
```suggestion
- [Stable variables](/internet-computer-programming-concepts/basic-memory-persistence/stable-variables.html) are preserved
```
I'd avoid using the work persist for the retention of stable variables to avoid confusion with orthogonal persistence, which, on the IC, refers to the ability of an actor to (transparently) retain its state across message invocations, in between upgrades. In traditional web apps, a service would need to explicitly restore and save its state from a  database. On the IC, the platform does this for you so the user has the illusion of memory changes persisting across message invokations, without any further programming.

Reply:
Got it!